baseplate/clients/redis_cluster.py

```diff
@@ -333,12 +333,19 @@ class ClusterRedisClient(config.Parser):
     See :py:func:`cluster_pool_from_config` for available configuration settings.
```

Comment:
🔕 on the other one you added some pydoc describing the params and some comments below, did you want to duplicate them?